packages/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -2,12 +2,12 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
-import { IMember, IServiceAudience } from "@fluidframework/fluid-static";
-import { IUser } from "@fluidframework/protocol-definitions";
-import { ITokenProvider } from "@fluidframework/routerlicious-driver";
+import { type ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
+import { type IMember, type IServiceAudience } from "@fluidframework/fluid-static";
+import { type IUser } from "@fluidframework/protocol-definitions";
+import { type ITokenProvider } from "@fluidframework/routerlicious-driver";
 // Re-export so developers can build loggers without pulling in core-interfaces
-export { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
+export type { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
```

Comment:
In line 6 we add `type` for each entry. I imagine this is equivalent, but would be nice to use one or the other everywhere.

Reply:
Thanks! This is how eslint likes to auto-fix, but I agree that individual entry level is better.

```suggestion
export { type ITelemetryBaseEvent, type ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
```